Add Dragino LPS8N Basics Station docs (#1340)
TheThingsIndustries/lorawan-stack-docs: doc/content/gateways/models/dragino-lps8/lbs.md

---
title: "Connect Dragino LPS8N with Lora Basics™ Station"
description: ""
---

This section contains instructions for connecting the Dragino LPS8N LoRaWAN® gateway to {{% tts %}} using [{{% lbs %}}]({{< ref "/gateways/concepts/lora-basics-station" >}}).

<!--more-->

Before proceeding with the next step, we assume you have set up internet access for the LPS8N using one of the following methods:

- Using the WAN port of the LPS8N
- As a WiFi client
- Using the built-in 4G modem

## Registering Gateway

> In the The Things Stack console, go to the 'Gateways' tab and click the '**Register Gateway**' button to register the gateway and make sure to select these three settings below:

Reviewer: ```suggestion
In the {{% tts %}} console, go to the **Gateways** tab and click the **Register Gateway** button to register the gateway and make sure to select these three settings below:
```

{{< figure src="../register-lbs.png" alt="" width="80%">}}

> When the keys popup shows after registering, download the keys, and click "I have downloaded the keys".

Reviewer: ```suggestion
When the keys popup shows after registering, download the keys, and click **I have downloaded the keys**.
```

{{< figure src="../register-lbs-keys.png" alt="" >}}

{{< note >}}
If you have already registered your gateway and forgot to check the boxes, or if you have been using UDP and want to switch to {{% lbs %}}, [create API keys manually](https://www.thethingsindustries.com/docs/gateways/concepts/lora-basics-station/cups/#create-separate-cups-and-lns-api-keys).
{{< /note >}}

### Configuring Gateway

In the Gateway's Web UI, navigate to **LoRaWAN > LoRaWAN** in the top menu.

On this page, configure the following settings:

- **Service Provider**: `The Things Network -- Basic Station`

> - **CUPS Server URI**: your server address with :443 (e.g. `https://eu1.cloud.thethings.network:443`)

Reviewer: ```suggestion
- **CUPS Server URI**: {{% tts %}} server address with port 443 (e.g. `https://eu1.cloud.thethings.network:443`)
```

- **CUPS Authorization Key**: The CUPS key you just downloaded.

> - **LNS Authorization Key**: the LNS key you just downloaded.

Reviewer: ```suggestion
- **LNS Authorization Key**: The LNS key you just downloaded.
```

> - **CUPS Certificate Authority**: Install a certificate by clicking on the `DEFAULT-CERTIFICATE` button.

Reviewer: ```suggestion
- **CUPS Certificate Authority**: Install a certificate by clicking on the `DEFAULT_CERTIFICATE` button.
```

---
title: "Connect Dragino LPS8N with Lora Basics™ Station"
description: ""
---

This section contains instructions for connecting the Dragino LPS8N LoRaWAN® gateway to {{% tts %}} using [{{% lbs %}}]({{< ref "/gateways/concepts/lora-basics-station" >}}).

> If you have not yet registered your gateway, follow [Registering Gateway](#registering-gateway)

Reviewer: You can also use links from here https://www.thethingsindustries.com/docs/gateways/concepts/lora-basics-station/ to compress this even more. For example, for creating keys part you can say just follow this section https://www.thethingsindustries.com/docs/gateways/concepts/lora-basics-station/cups/#create-separate-cups-and-lns-api-keys
